changelogs/api-management-service.md

```diff
@@ -4,7 +4,7 @@
 
 A regular Azure API Management service update was started on . This release will continue to rollout through May 2023.
 
-### New features, fixes, and improvement
+### New features, fixes, and improvements
 
 1. The Azure AD authentication in Configuration API v2 has been updated.
```

Comment:
In points 1 & 2, can you explain what changed to the customers and how they can start using it? "has been updated" is not meaningful.

Reply:
No longer relevant, see #38